microcosm_flask/swagger/parameters/base.py

```diff
@@ -73,6 +76,10 @@ def parse_description(self, field: Field) -> Optional[str]:
         Parse the description for the field, if any.
 
         """
+        if "metadata" in field.metadata:
+            metadata = field.metadata.get("metadata")
+            if metadata:
+                return metadata.get("description")
```

Comment:
Nit: possible simplification

```suggestion
        metadata = field.metadata.get("metadata", {})
        if metadata:
            return metadata.get("description")
```

Reply:
Also, how do we get `field.metadata.metadata.description`? Does this happen?

Reply:
It happens for the fields of marshmallow `Schema`, but I'm not really sure why as we don't nest the metadata ourselves. I handled both cases to be defensive.